FlatBuffersSwift/FlatBuffersBuilder.swift

```diff
@@ -172,51 +237,78 @@ public final class FBBuilder {
         objectStart = Int32(cursor)
     }
     
+    /**
+     Append an offset into the buffer for the currently open object
+     
+     - parameters:
+         - propertyIndex: The index of the property to update
+         - offset: The offsetnumber of properties we will update
+
+     - Returns: The current cursor position (Note: What is the use case of the return value?)
+     */
     @discardableResult
-    public func addPropertyOffsetToOpenObject(propertyIndex : Int, offset : Offset) throws -> Int{
+    public func append(propertyIndex : Int, offset : Offset) throws -> Int{
```

Comment:
also here `append`feels wrong. maybe `insert(offset:toStartedObjectAt:)` is very long but describes the function in the best possible way I think.